### PR TITLE
Draft. GB-8 Backend. Django backend, add Model Structure.

### PR DESCRIPTION
**Pull request by luckyman0333**
https://github.com/luckyman0333/ExpenseProject/issues/8

### CODE REVIEW

File: expense_manager/expense_manager/settings.py [hunk 1]
```diff
@@ -54,7 +55,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [BASE_DIR / 'templates'],
```
Comment (SerhiiRI) [inline expense_manager/expense_manager/settings.py]: according to commit files , config line `[BASE_DIR / 'templates']` your PR doesn't include any 'template' folder. So  or you 
1) comment this line and set `[]` empty list 
or 
2) add template dir. 
This  decision is up to you

File: expense_manager/expense_manager/urls.py [hunk 2]
```diff
@@ -15,8 +15,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
```
Comment (SerhiiRI) [inline expense_manager/expense_manager/urls.py]: Cause you have to fix previous my comment, please remove unused `, include` import(or comment it)

File: expense_manager/expense_manager/settings.py (hunk 1 above)
Comment (luckyman0333) [reply expense_manager/expense_manager/settings.py]: i deciede 1 variant. set [] empty list. this line will be added when templates will maked

File: expense_manager/expense_manager/urls.py (hunk 2 above)
Comment (luckyman0333) [reply expense_manager/expense_manager/urls.py]: done

**Pull request merged**